Mathlib/Order/Filter/Defs.lean

```diff
@@ -296,17 +296,14 @@ def comap (m : α → β) (f : Filter β) : Filter α where
   inter_sets := fun ⟨a', ha₁, ha₂⟩ ⟨b', hb₁, hb₂⟩ =>
     ⟨a' ∩ b', inter_mem ha₁ hb₁, inter_subset_inter ha₂ hb₂⟩
 
-/-- Product of filters. This is the filter generated by cartesian products
-of elements of the component filters. -/
-protected def prod (f : Filter α) (g : Filter β) : Filter (α × β) :=
```

Comment:
Should I leave it as a deprecated definition?

Reply:
I would say: yes, out of principle. Dealing with deprecations is *much* nicer than dealing with removed definitions.